src/xunit.netcore.extensions/Discoverers/ConditionalTestDiscoverer.cs

```diff
@@ -135,6 +105,41 @@ internal class ConditionalTestDiscoverer
             return testCases;
         }
 
+        internal static bool EvaluateParameter(object[] conditionArguments)
```

Comment:
If we're not using this method here, then just move it to ConditionalDiscoverer and just leave CheckInputSkipExcecution here.

Reply:
Sounds good. Will fix.